src/k2/cmd/txbench/k23sibench_client.cpp

```diff
@@ -153,6 +153,7 @@ class Client {
         })
         .finally([this] {
             K2LOG_I(log::txbench, "Done with benchmark");
+            AppBase().stop(0);
```

Comment:
We need this app to stay up so that we can collect its last few metric datapoints. Unless you need it to shutdown by itself, you should probably leave it as-is